browser/net/brave_network_audit_allowed_lists.h

```diff
@@ -3,20 +3,20 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#ifndef BRAVE_BROWSER_NET_BRAVE_NETWORK_AUDIT_WHITELISTS_H_
-#define BRAVE_BROWSER_NET_BRAVE_NETWORK_AUDIT_WHITELISTS_H_
+#ifndef BRAVE_BROWSER_NET_BRAVE_NETWORK_AUDIT_LISTS_H_
+#define BRAVE_BROWSER_NET_BRAVE_NETWORK_AUDIT_LISTS_H_
```

Comment:
This should be `BRAVE_BROWSER_NET_BRAVE_NETWORK_AUDIT_ALLOWED_LISTS_H_` now

Reply:
Oops, thanks for the additional review!